src/TheAlgorithms.jl

```diff
@@ -67,7 +67,7 @@ export sum_gp
 export surfarea_cube
 export surfarea_sphere
 export trapazoidal_area
-export eratosthenes
+export verlet_integration
```

Comment:
this too has caused the code cove issue because of the deleted export, thus failed tests